src/Avalonia.Controls/MenuItem.cs

```diff
@@ -194,6 +202,19 @@ public object Icon
             set { SetValue(IconProperty, value); }
         }
 
+        /// <summary>
+        /// Gets or sets the input gesture that will be displayed in the menu item.
+        /// </summary>
+        /// <remarks>
+        /// Setting this property does not cause the input gesture to be handled by the menu item,
+        /// it simply displays the gesture text in the menu.
+        /// </remarks>
+        public object InputGestureText
```

Comment:
I guess it was supposed to be `string` and not `object` ?

Reply:
D'oh! Not sure how that happened.

Reply:
Fixed